src/devices.rs

```diff
@@ -1,49 +1,36 @@
-use std::process::{Command, Stdio};
+use failure::Error;
 
-use failure::{Error, err_msg};
+use adb_command::AdbCommand;
 
 #[derive(Debug, PartialEq, Clone)]
 pub struct Device {
     pub id: String,
-    pub name: String,
+    pub details: String,
 }
 
 impl Device {
     pub fn list_devices() -> Result<Vec<Device>, Error> {
-        let child = Command::new("adb")
-            .arg("devices")
-            .stdout(Stdio::piped())
-            .spawn()?;
+        let output = AdbCommand::command("devices".to_string())
+            .with_args(vec!["-l".to_string()])
```

Comment:
maybe we need a `with_arg` which takes just a string instead of a vec

Reply:
Thought it is only one command where we need just one parameter but there are more (two) so yes, will fix that 👍